channel.c

```diff
@@ -152,17 +152,7 @@ void iio_channel_init_finalize(struct iio_channel *chn)
 		return;
 
 	mod++;
-
-	for (i = 0; i < ARRAY_SIZE(modifier_names); i++) {
-		if (!modifier_names[i])
-			continue;
-		len = strlen(modifier_names[i]);
-		if (strncmp(modifier_names[i], mod, len) != 0)
```

Comment:
the only thing here [and where we'd need to take care] is the fact that `find_channel_modifier()` also checks if `(s[len] == '\0' || s[len] == '_')`
that [in itself] could be fine for this change; though i am not completely sure;

Reply:
You're right, We can find numbers as well behind the modifier.

Reply:
Actually, there is not number behind the modifier: If I understand correctly, the channel name is:
[in|out]_[type]{channel number}_[modifier]{_extra_modifier}